app/__init__.py

```diff
@@ -113,4 +119,11 @@ def projects():
             "URL": "https://github.com/jiechenmc/poke_data_explorer",
             "status": "incomplete"
         },
-    ])
\ No newline at end of file
+    ])
+
+# Wrapping these commands so tests/test_db.py can import TimelinePost without 
+# the db and app trying to connect and run.
```

Comment:
This should not be needed, you can either add `os.environ['TESTING'] = 'true'` before importing app or move TimelinePost model to a different file.